app/models/card.rb

```diff
@@ -20,6 +21,10 @@ class Card < ActiveRecord::Base
     order("RANDOM()").first
   }
 
+  scope :current_deck, -> {
+    joins(:deck).where('current = ?', true)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.